src/components/ChangeCourtSize/CourtList.tsx

```diff
@@ -1,30 +1,36 @@
 const courtList = [
   {
+    courtId: "62c432cfb8a9c5f61f03831f",
     courtSizeName: "510 m² Pro Court",
     courtSizeDetails: "(17 m × 30 m)",
```

Comment:
We may do not need courtSizeName and courtSize details here. Cos the name we can get from courtSizeSlice and the numbers are variables( border changes, the number changes).  Same as below.

Reply:
<img width="260" alt="Screen Shot 2022-07-06 at 4 35 50 pm" src="https://user-images.githubusercontent.com/84214439/177484743-8ee3263a-82aa-4379-b140-5776b46f2965.png">
I understand the courtSizeName and courtSizeDetails are used in the string showing on the picture. We may change the logic, using concatenated string by adding the data variables with the rest characters to show the content above.